### PR TITLE
Docs: Add favicon everywhere

### LINKED ISSUE #15: Add a favicon to the documentation
### Description

Our documentation doesn't have any favicon files for now.

To close this issue, a favicon file (or files) should be added to all pages of the documentation.

There could be extra things to do for https://ods-charts.netlify.app/api/.

Don't forget to add the favicon file (or files) to our https://github.com/Orange-OpenSource/ods-charts/blob/main/NOTICE.txt.

### Sub-tasks

- [x] Add favicons images in the repository and mention them in a `NOTICE.txt`
- [x] Add favicons to the documentation
- [ ] Add favicons to the `/api` (we might need something like https://www.npmjs.com/package/typedoc-plugin-extras as an extra plugin to do that)

### PR DESCRIPTION
**Pull request by louismaximepiton**
_Note: Please transform `- [ ]` into `- (NA)` in the description when things are not applicable_

### Related issues

Related to #15.

### Description

Adding a favicon file and use it everywhere.

### Motivation & Context

The documentation had no favicon.

### Types of change

- New feature (non-breaking change which adds functionality)

### Test checklist

Please check that the following tests projects are still working:

- [x] `docs/examples`
- [ ] `test/angular-ngx-echarts`
- [ ] `test/angular-tour-of-heroes`
- [ ] `test/html`
- [ ] `test/react`
- [ ] `test/vue`
- [ ] `test/examples/bar-line-chart`
- [ ] `test/examples/single-line-chart`
- [ ] `test/examples/timeseries-chart`

### CODE REVIEW

**Comment by julien-deramond** [comment]
When I go to https://deploy-preview-87--ods-charts.netlify.app/examples/single-line-chart or https://deploy-preview-87--ods-charts.netlify.app/examples, I can't see any favicon in the browser and there's a `GET https://deploy-preview-87--ods-charts.netlify.app/favicon.ico 404 (Not Found)` error in the console.

If possible, by treating this issue, we should also provide the other formats of favicons. For example, as it's done for Boosted:

Screenshot 2024-01-30 at 07 31 32

File: docs/images/favicons/manifest.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Boosted",
+  "short_name": "Boosted",
```
Comment (julien-deramond) [inline docs/images/favicons/manifest.json:3]:
```suggestion
  "short_name": "ODS Charts",
```

File: docs/images/favicons/manifest.json [hunk 2]
```diff
@@ -0,0 +1,20 @@
+{
+  "name": "Boosted",
```
Comment (julien-deramond) [inline docs/images/favicons/manifest.json:2]:
```suggestion
  "name": "Orange Design System Charts",
```

**Comment by louismaximepiton** [comment]
Should work properly on examples side.
I had to introduce relative path since the URL differs between local and netlify. Should I fix this as well ?
Idk if you want it on API side as well but we might need https://www.npmjs.com/package/typedoc-plugin-extras as an extra plugin ?

**Review by julien-deramond — APPROVED** [review]
Thanks for the PR @louismaximepiton 
I've just made minor modifications in the `NOTICE.txt` file via https://github.com/Orange-OpenSource/ods-charts/pull/87/commits/96c8de0c8b71f28198a9c9974ec4d52caef4f816 and https://github.com/Orange-OpenSource/ods-charts/pull/87/commits/a0896ee0f96801a5d28827995fe23552f871df5a.
I noticed that `<meta name="theme-color" content="#000">` was missing; added via https://github.com/Orange-OpenSource/ods-charts/pull/87/commits/48ed5a3c5922bd830819db5bccef388b4ed48791.

> Should work properly on examples side.
> I had to introduce relative path since the URL differs between local and netlify. Should I fix this as well ?

Let's let it as you've done it in the PR. It works well in Netlify deployment pages but also locally. We'll revisit later on if needed :)

> Idk if you want it on API side as well but we might need https://www.npmjs.com/package/typedoc-plugin-extras as an extra plugin ?

Let's do that in a separate PR. Based on that, I've removed the link to the issue (this PR won't close it) and I'll create a sub-list of tasks in the issue to show that the only remaining thing will be that: adding favicons to the `/api` URL.

LGTM so :shipit:

**Pull request merged**